itest/litd_mode_integrated_test.go

```diff
@@ -787,11 +838,19 @@ func runUIPasswordCheck(t *testing.T, hostPort, tlsCertPath, uiPassword string,
 	require.NoError(t, err)
 	defer rawConn.Close()
 
-	// Make sure that a call without any metadata results in an error.
+	// Make sure that a call without any metadata results in an error unless
+	// this is a call that is allowed to be un-authenticated in which case
+	// we expect it to succeed.
 	_, err = makeRequest(ctxt, rawConn)
-	if disabled {
+	switch {
+	case disabled:
 		require.ErrorContains(t, err, disabledErr)
-	} else {
+	case noAuth:
+		require.NoError(t, err)
```

Comment:
Should we assert the success content here before returning?

Reply:
jip - good catch 👍  also updated the `runGRPCWebAuthTest` test to properly check success content